aspnet/mvc/models/validation.rst

```diff
@@ -1,8 +1,196 @@
-.. include:: /../common/stub-topic.txt
+Model Validation 
+================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-|stub-icon| Model Validation
-============================
+In this article:
 
-.. include:: /../common/stub-notice.txt
+.. contents:: Sections
+  :local:
+  :depth: 1
 
-.. _issue: https://github.com/aspnet/Docs/issues/116
+Introduction to model validation
+--------------------------------
+Before an app stores data in a database, the app must validate the data. Data must be scrubbed for security, verified that it is appropriately formatted by type and size, and it must conform to your rules. Validation is necessary although it can be redundant and tedious to implement. In MVC, validation happens on both the client and server. 
+
+With data annotations in MVC apps you can validate data by customizing models so they can enforce business rules against models by applying attributes to model properties. These attributes generate validation code, thereby reducing the amount of code you must write. Fortunately, .NET has abstracted validation into the data annotations, often reducing validation to a single line of code.
```

Comment:
I got confused reading this paragraph. Suggestions:
- Start w/ the "Fortunately, ..." sentence since it ties into the previous paragraph.
- Reduce the number of subjects in the current first sentence or make them more clear. E.g. who / what applies attributes to model properties?
- "These attributes generate validation code" -> "These attributes contain validation code".